src/Lykke.Sdk/LykkeWebHostFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using System;
+using System.IO;
+
+namespace Lykke.Sdk
+{
+    /// <summary>
+    /// This class creates IWebHostBuilder which is used in LykkeStarter.
+    /// </summary>
+    public class LykkeWebHostFactory
```

Comment:
Let's make this class static

Reply:
Better to keep it not static. This will allow us to inject a dependency to the factory if needed.

Reply:
SUch functionality is hardly needed. Besides we have LykkeWebHostFactoryOptions to inject/override whatever we need.